lua/autorun/server/bounties.lua

```diff
@@ -0,0 +1,78 @@
+
+hook.Add("PlayerDeath","PlayerDeath",function(ply,infl,atk)
+	local bounty = tonumber(ply:GetPData("bounty",0))
+	if bounty > 0 and ply != atk and type(atk) == "Player" then
+		atk:PS_GivePoints(bounty)
+		BotSayGlobal("[edgy]"..atk:Nick().." [fbc]has claimed [gold]"..ply:Nick().."'s [fbc]bounty of [rainbow]"..bounty.." [fbc]points!")
+		ply:SetPData("bounty",0)
+	end
+end)
+
+function FindPlayer(plyname)
```

Comment:
Use my utility function instead of this: `PlyCount(name) -> player, count`
player is one player that matches, or nil if none matches. count is the total number of matches so if count>1 then you can err